src/AqueductV1Auction.sol

```diff
@@ -133,14 +140,18 @@ contract AqueductV1Auction is IAqueductV1Auction {
         // just swap the swap amount, bid/fee will be sent in executeWinningBid()
         // swapped funds will be locked until executeWinningBid() is called
         uint256 amountOut = swap(token, pair, swapAmount, reserve0, reserve1, token0);
+        if (amountOut < amountOutMin) revert AUCTION_UNDER_MIN_AMOUNT_OUT();
 
         // update auction
         auction.token = token;
         auction.winningBid = bid;
```

Comment:
Can't remember if this still needed doing but based on your comment in TG:
> One thing I did notice here is that we are tracking auction.winningBid by setting it to 'bid', as opposed to 'bidValue'

Should this be `auction.winningBid = bidValue;`?